ddtrace/internal/ci_visibility/encoder.py

```diff
@@ -35,11 +34,7 @@ class CIVisibilityEncoderV01(BufferedEncoder):
     ALLOWED_METADATA_KEYS = ("language", "library_version", "runtime-id", "env")
     PAYLOAD_FORMAT_VERSION = 1
     TEST_SUITE_EVENT_VERSION = 1
-    # Change to 1 to allow unittest tests without module or session spans
-    if config._ci_visibility_unittest_enabled:
-        TEST_EVENT_VERSION = 1
-    else:
-        TEST_EVENT_VERSION = 2
+    TEST_EVENT_VERSION = 2
```

Comment:
With this change ``TEST_EVENT_VERSION`` is always 2. Is this documented somewhere? Should we even call this out?

Reply:
I do not think we should call this out, `TEST_EVENT_VERSION` was temporarily set to 1 due to the `unittest` beta. This is now reverted to pre-unittest branches.